stacker/lookups/handlers/output.py

```diff
@@ -17,11 +18,22 @@ def handler(value, provider=None, context=None, fqn=False, **kwargs):
         fqn (bool): boolean for whether or not the
             :class:`stacker.context.Context` should resolve the `fqn` of the
             stack.
+        rfqn (bool): boolean for whether or not the
+            :class:`stacker.context.Context` should resolve the `fqn` of the
+            stack prefixed by the namespace variable
 
     Returns:
         str: output from the specified stack
 
     """
+
+    if rfqn and context.environment.get('namespace'):
+            value = "%s%s%s" % (
+                    context.environment['namespace'],
+                    context.environment.get('namespace_delimiter')
```

Comment:
There should never be a time that the delimiter isn't set as far as I know, but if you can think of a time when that would happen you would better express this as:

`context.environment.get('namespace_delimiter', '-')`